sixtyfps_runtime/corelib/Cargo.toml

```diff
@@ -15,7 +15,13 @@ path = "lib.rs"
 [features]
 # Allow the viewer to query at runtime information about item types
 rtti = []
+# Expose C ABI
 ffi = []
+# Use the standard library
+std = []
```

Comment:
Is it intentional not to enable the std features of euclid, etc. here? (Just curious)

Reply:
yes, i don't believe we need them, at least it compiles without it

Reply:
That makes sense - great :)